tutor/src/screens/grading-templates/editors.js

```diff
@@ -97,6 +97,13 @@ const StyledTextInput = styled(Field).attrs({
   border-radius: 4px;
   border: 1px solid ${colors.forms.borders.light};
   font-size: 1.4rem;
+  /** styling errors when template name is invalid */
+  background: ${props => props.hasError ? '#fbe8eA' : '#FFFFFF'};
+  color: ${props => props.hasError ? 'red' : 'black'};
+  font-weight: ${props => props.hasError ? 500 : 400};
```

Comment:
I think the Invision spec has this as the default weight instead of bold?

Reply:
Oh we can probably bump the font size down to 1.2rem as well.

Reply:
oh yeah. I don't know why at first it looked a little bolder to me. thanks!